sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/StateStoreSuite.scala

```diff
@@ -814,11 +818,53 @@ class StateStoreSuite extends StateStoreSuiteBase[HDFSBackedStateStoreProvider]
   }
 }
 
+class StateStoreCompatibleSuite extends StreamTest with StateStoreCodecsTest {
```

Comment:
Apache Spark uses `XXXCompatibilitySuite` instead of `XXXCompatibleSuite`. Could you rename this sure?

Reply:
Sure.